lib/ecto_interval.ex

```diff
@@ -69,9 +69,15 @@ if Code.ensure_loaded?(Postgrex) do
     end
 
     @impl true
-    def load(%{months: months, days: days, weeks: weeks, secs: secs}) do
+    def load(%{months: months, days: days, secs: secs}) do
```

Comment:
Don't we want to consume weeks?

Reply:
Postgres intervals are only stored using months, days, secs. We then translate this with weeks and years